src/main/java/be/seeseemelk/mockbukkit/entity/EnderPearlMock.java

```diff
@@ -0,0 +1,47 @@
+package be.seeseemelk.mockbukkit.entity;
+
+import be.seeseemelk.mockbukkit.ServerMock;
+import com.google.common.base.Preconditions;
+import org.bukkit.Material;
+import org.bukkit.entity.EnderPearl;
+import org.bukkit.entity.EntityType;
+import org.bukkit.inventory.ItemStack;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.UUID;
+
+public class EnderPearlMock extends ThrowableProjectileMock implements EnderPearl
+{
+	private @NotNull ItemStack item = new ItemStack(Material.ENDER_PEARL);
+
+	/**
+	 * Constructs a new {@link EnderPearlMock} on the provided {@link ServerMock} with a specified {@link UUID}.
+	 *
+	 * @param server The server to create the entity on.
+	 * @param uuid   The UUID of the entity.
+	 */
+	public  EnderPearlMock(@NotNull ServerMock server, @NotNull UUID uuid)
+	{
+		super(server, uuid);
+	}
+
+	@Override
+	public @NotNull ItemStack getItem()
+	{
+		return this.item;
```

Comment:
```suggestion
		return new ItemStack( this.item );
```
This should also return a copy of the stored item.